unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,638 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+######################################
+# This set of tests is valid for any device that supports the HDR feature #
+######################################
+
+# test:device D400* !D457
+
+import pyrealsense2 as rs
+from rspy import test, log
+import time
+
+# HDR CONFIGURATION TESTS
+with test.closure("HDR Config - default config", "[hdr][live]"):
+    ctx = rs.context()
+    device = test.find_first_device_or_exit()
+    if ctx is not None:
+        devices_list = ctx.query_devices()
+        device_count = devices_list.size()
+        if device_count != 0:
+            depth_sensor = device.first_depth_sensor()
+
+            if depth_sensor and depth_sensor.supports(rs.option.hdr_enabled):
+                exposure_range = depth_sensor.get_option_range(rs.option.exposure)
+                gain_range = depth_sensor.get_option_range(rs.option.gain)
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 1)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+                depth_sensor.set_option(rs.option.sequence_id, 1)
+                test.check(depth_sensor.get_option(rs.option.sequence_id) == 1)
+                exp = depth_sensor.get_option(rs.option.exposure)
+                test.check(depth_sensor.get_option(rs.option.exposure) == exposure_range.default - 1000)  # w/a
+                test.check(depth_sensor.get_option(rs.option.gain) == gain_range.default)
+
+                depth_sensor.set_option(rs.option.sequence_id, 2)
+                test.check(depth_sensor.get_option(rs.option.sequence_id) == 2)
+                test.check(depth_sensor.get_option(rs.option.exposure) == exposure_range.min)
+                test.check(depth_sensor.get_option(rs.option.gain) == gain_range.min)
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 0)
+                enabled = depth_sensor.get_option(rs.option.hdr_enabled)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 0)
+        else:
+            log.e("No device was found - skipping test")
```

Comment:
Can be removed as 
`test.find_first_device_or_exit()`
Will handle that.
Same for all tests below